### DISCUSSION TITLE
Student developer pack benefits take a long time to be available

### DISCUSSION THREAD

**Discussion by jeremyelalouf** [community/community#131048] [GitHub Education]
### Hi everyone,

Hi anyone, would you think the I can get help on the status of my demand of the Github Developer Student pack ?

**Comment by davevad93** [comment]
Hi @jeremyelalouf ,

Student benefits can take a while to be available, the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) states that If you haven't received your education benefits within 5 days of the approval status to open a discussion here so that someone from the GitHub Education staff can assist you. In general, people receive their benefits within a few days but there are cases where you have to wait a couple of weeks and this usually depends on different factors such as the volume of applications.
